packages/training/Makefile

```
@@ -471,9 +480,49 @@ train-cloud-online: db-up db-migrate bridge-check
+
+# Docker-based benchmark (recommended)
+local-benchmark:
+	@echo "$(CYAN)Running Docker benchmark...$(RESET)"
+	cd $(LOCAL_DIR) && ./benchmark.sh --model $(BENCHMARK_MODEL) $(if $(SCENARIO),--scenario $(SCENARIO),)
+
+local-benchmark-quick:
+	@echo "$(CYAN)Running quick Docker benchmark...$(RESET)"
+	cd $(LOCAL_DIR) && ./benchmark.sh --model $(BENCHMARK_MODEL) --quick $(if $(SCENARIO),--scenario $(SCENARIO),)
+
+# Native benchmark (requires full monorepo setup with bun)
+benchmark:
```

checkmake (0.2.2)

[warning] Target body for "benchmark" exceeds allowed length of 5 (6).

(maxbodylength)